### PR TITLE
RESOURCE-198 SQL SSL Cert

### PR DESCRIPTION
**Pull request by sa-progress**
Signed-off-by: sa-progress <samir.anand@progress.com>

### Description

Please describe what this change achieves. Ensure you have read the [Contributing to InSpec GCP](https://github.com/inspec/inspec-gcp/CONTRIBUTING.md) document before 
submitting.

### Issues Resolved

List any existing issues this PR resolves, or any Discourse or StackOverflow discussion that's relevant.

Please ensure commits have been signed-off for the Developer Certificate of Origin. See <https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco>

### CODE REVIEW

**Review by soumyo13 — APPROVED** [review]

File: test/integration/verify/controls/google_sql_ssl_certs.rb [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+#
+#     Please read more about how to change this file in README.md and
+#     CONTRIBUTING.md located at the root of this package.
+#
+# ----------------------------------------------------------------------------
+
+title 'Test GCP google_sql_ssl_certs resource.'
+
+gcp_project_id = attribute(:gcp_project_id, default: 'gcp_project_id', description: 'The GCP project identifier.')
+sql_connect = attribute('sql_connect', default: {
```
Comment (soumyo13) [inline test/integration/verify/controls/google_sql_ssl_certs.rb:18]:
```suggestion
gcp_project_id = attribute(:gcp_project_id, value: 'gcp_project_id', description: 'The GCP project identifier.')
sql_connect = attribute('sql_connect', value: {
```

File: docs/resources/google_sql_ssl_certs.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+---
+title: About the google_sql_ssl_certs resource
+platform: gcp
+---
+
+## Syntax
+A `google_sql_ssl_certs` is used to test a Google SslCert resource
```
Comment (soumyo13) [inline docs/resources/google_sql_ssl_certs.md:7]:
```suggestion
A `google_sql_ssl_certs` is used to test a Google SslCerts resource
```

**Review by soumyo13 — CHANGES_REQUESTED** [review]

**Comment by soumyo13** [comment]
Please make the following changes as default is depreciated.

File: test/integration/verify/controls/google_sql_ssl_certs.rb [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+#     changes will be clobbered when the file is regenerated.
+#
+#     Please read more about how to change this file in README.md and
+#     CONTRIBUTING.md located at the root of this package.
+#
+# ----------------------------------------------------------------------------
+
+title 'Test GCP google_sql_ssl_certs resource.'
+
+gcp_project_id = attribute(:gcp_project_id, value: 'gcp_project_id', description: 'The GCP project identifier.')
```
Comment (soumyo13) [inline test/integration/verify/controls/google_sql_ssl_certs.rb:17]:
```suggestion
gcp_project_id = attribute(:gcp_project_id, default: 'gcp_project_id', description: 'The GCP project identifier.')
```

File: test/integration/verify/controls/google_sql_ssl_certs.rb [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+#
+#     Please read more about how to change this file in README.md and
+#     CONTRIBUTING.md located at the root of this package.
+#
+# ----------------------------------------------------------------------------
+
+title 'Test GCP google_sql_ssl_certs resource.'
+
+gcp_project_id = attribute(:gcp_project_id, default: 'gcp_project_id', description: 'The GCP project identifier.')
+sql_connect = attribute('sql_connect', value: {
```
Comment (soumyo13) [inline test/integration/verify/controls/google_sql_ssl_certs.rb:18]:
```suggestion
sql_connect = attribute('sql_connect', default: {
```

File: test/integration/verify/controls/google_sql_ssl_cert.rb [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+#
+#     Please read more about how to change this file in README.md and
+#     CONTRIBUTING.md located at the root of this package.
+#
+# ----------------------------------------------------------------------------
+
+title 'Test GCP google_sql_ssl_cert resource.'
+
+gcp_project_id = attribute(:gcp_project_id, value: 'gcp_project_id', description: 'The GCP project identifier.')
+sql_connect = attribute('sql_connect', value: {
```
Comment (soumyo13) [inline test/integration/verify/controls/google_sql_ssl_cert.rb:18]:
```suggestion
gcp_project_id = attribute(:gcp_project_id, default: 'gcp_project_id', description: 'The GCP project identifier.')
sql_connect = attribute('sql_connect', default: {
```

File: test/integration/verify/controls/google_sql_ssl_cert.rb [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+#
+#     Please read more about how to change this file in README.md and
+#     CONTRIBUTING.md located at the root of this package.
+#
+# ----------------------------------------------------------------------------
+
+title 'Test GCP google_sql_ssl_cert resource.'
+
+gcp_project_id = attribute(:gcp_project_id, default: 'gcp_project_id', description: 'The GCP project identifier.')
+sql_connect = attribute('sql_connect', default: {
```
[the remark this answers is not in the document]
Comment (sa-progress) [reply test/integration/verify/controls/google_sql_ssl_cert.rb:18]: Please revert back.Can change it manually.

**Review by soumyo13 — APPROVED** [review]

**Pull request merged**